scripts/seed_demo_user.py

```
@@ -0,0 +1,310 @@
+    user = created.user if hasattr(created, "user") else created
+    user_id = user.id if hasattr(user, "id") else user["id"]
+    print(f"  ok: created auth user {user_id}")
+    return user_id
+
+
+def wipe_user_data(sb: Client, user_id: str) -> None:
+    for table in USER_OWNED_TABLES_TO_WIPE:
+        try:
+            sb.table(table).delete().eq("user_id", user_id).execute()
+        except Exception as exc:
+            print(f"  warn: failed to wipe {table}: {exc}", file=sys.stderr)
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)